pandas/tests/io/excel/test_openpyxl.py

```diff
@@ -0,0 +1,124 @@
+import pytest
+
+import pandas.util._test_decorators as td
+
+from pandas import DataFrame
+from pandas.util.testing import ensure_clean
+
+from pandas.io.excel import ExcelWriter, _OpenpyxlWriter
+
+
+@td.skip_if_no('openpyxl')
```

Comment:
use a pytest.importorskip

also I would not use classes unless you have separate discrete cases